config-example/values.env

```diff
@@ -9,12 +9,11 @@ export GENESIS_FORK_VERSION="0x10000000"
 export ALTAIR_FORK_VERSION="0x20000000"
 export BELLATRIX_FORK_VERSION="0x30000000"
 export CAPELLA_FORK_VERSION="0x40000000"
-export CAPELLA_FORK_EPOCH="100000"
 export DENEB_FORK_VERSION="0x50000000"
-export DENEB_FORK_EPOCH="200000"
+export DENEB_FORK_EPOCH="5"
```

Comment:
I wouldn't change Deneb defaults, capella genesis doesn't imply we have to test deneb